apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerCommandTests.cs

```diff
@@ -85,5 +91,100 @@ public void CloneUpdateWithParameters()
             Assert.True(command.Parameters.SequenceEqual(command2.Parameters));
         }
 
+        /*
```

Comment:
Tests are commented out for now since I ran out of time to setup the mock. Will complete this next week.